[31merror[39m:   uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1902:16)
    at listenInCluster (node:net:1959:12)
    at Server.listen (node:net:2061:7)
    at Function.listen (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:25:5)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
[31merror[39m:   uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1902:16)
    at listenInCluster (node:net:1959:12)
    at Server.listen (node:net:2061:7)
    at Function.listen (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:25:5)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
[31merror[39m:   uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1902:16)
    at listenInCluster (node:net:1959:12)
    at Server.listen (node:net:2061:7)
    at Function.listen (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:25:5)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
[31merror[39m:   uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1902:16)
    at listenInCluster (node:net:1959:12)
    at Server.listen (node:net:2061:7)
    at Function.listen (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:25:5)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
[31merror[39m:   uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1902:16)
    at listenInCluster (node:net:1959:12)
    at Server.listen (node:net:2061:7)
    at Function.listen (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:25:5)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
[31merror[39m:   uncaughtException: path is not defined
ReferenceError: path is not defined
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:20:24)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:166:5)
    at node:internal/main/run_main_module:30:49
[31merror[39m:   uncaughtException: path is not defined
ReferenceError: path is not defined
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:20:24)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:166:5)
    at node:internal/main/run_main_module:30:49
[31merror[39m:   uncaughtException: path is not defined
ReferenceError: path is not defined
    at Object.<anonymous> (/home/willchakur/Documentos/devWillChakur/ExchangeNow_vReact/backend/server.js:20:24)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:166:5)
    at node:internal/main/run_main_module:30:49
